stubs/icalendar/@tests/stubtest_allowlist.txt

```diff
@@ -1,9 +1,23 @@
 # Command line app
-icalendar\.cli
+icalendar.cli
 
 # Tests
 icalendar\.tests(\..*)?
 
 # Methods that use `int` to mean `bool`.
-icalendar\.cal\.Component\.get_inline
-icalendar\.cal\.Component\.set_inline
+icalendar.Component.get_inline
+icalendar.Component.set_inline
+icalendar.cal.Component.get_inline
+icalendar.cal.Component.set_inline
+
+# Runtime __all__ has extra items that don't exist.
```

Comment:
Is there a bug reported to the runtime package about this?

Reply:
Done now: collective/icalendar#715.